app/src/scripts/dataset/run/index.js

```diff
@@ -86,67 +91,6 @@ class JobAccordion extends React.Component {
     )
   }
 
-  _results(run, type) {
-    if (run[type] && run[type].length > 0) {
```

Comment:
Looks like we are losing the conditional to test for results in the refactor. This is causing the results according to display when job is still in pending.
![image](https://user-images.githubusercontent.com/5668826/34182714-4a940982-e4cc-11e7-88fc-22a873f99af2.png)

Also, I think it might be causing a crash when a job has no results 
![image](https://user-images.githubusercontent.com/5668826/34182751-6526a8c2-e4cc-11e7-8151-42ca68384a8f.png)

Reply:
![image](https://user-images.githubusercontent.com/5668826/34182821-acc81ad0-e4cc-11e7-8158-e28c718618f0.png)

this is probably a more useful error trace